OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 2

# Description
Lint fix for Reports component, this is phase 2.

## Files changed

- PeopleTableDetails.jsx
- ProjectTable.jsx
- ReportTableSearchPanel.jsx
- TeamTable.jsx
- TimeEntriesViz.jsx
- index.js

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Go to Reports,  Check functionality is the same as before.
- Check all the functionality of the report is the same as before, and see if any bugs have appeared.(Please record a video)

Optional:
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- `cd src/components/Reports`
  - Run `npx eslint `for the files: PeopleTableDetails.jsx  ProjectTable.jsx  ReportTableSearchPanel.jsx  TeamTable.jsx  TimeEntriesViz.jsx  index.js

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you f**ocus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

Screenshot:
<img width="1026" alt="CleanShot 2023-09-29 at 22 05 21@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/78afdc99-e652-4c95-88f7-53ae2f80b17b">


Please ignore the reports.jsx error, I will fix them later.
